core-bundle/contao/classes/Theme.php

```diff
@@ -320,7 +320,7 @@ protected function compareThemeFiles($arrFiles, $arrDbFields)
 
 			foreach ($exampleWebsites as $exampleWebsite)
 			{
-				$return .= '<option value="' . htmlspecialchars($exampleWebsite) . '">' . htmlspecialchars($exampleWebsite) . '</option>';
+				$return .= '<option value="' . htmlspecialchars($exampleWebsite, ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML5) . '">' . htmlspecialchars($exampleWebsite, ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML5) . '</option>';
```

Comment:
@ausi Is there a particular reason why we don‘t use `StringUtil::specialchars()` here and for the request token? The only difference is the double encoding, which `StringUtil::specialchars()` does not do by default. But that should not matter in these cases, right?